includes/Admin/Google_Product_Category_Field.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Admin;
+
+defined( 'ABSPATH' ) or exit;
+
+/**
+ * Google product category field.
+ *
+ * @since 2.1.0-dev.1
+ */
+class Google_Product_Category_Field {
+
+
+	/** @var string the WordPress option name where the full categories list is stored */
+	const OPTION_GOOGLE_PRODUCT_CATEGORIES = 'wc_facebook_google_product_categories';
+
+
+	/**
+	 * Instantiates the JS handler for the Google product category field.
+	 *
+	 * @internal
```

Comment:
Why internal on these methods? These will be used externally by both the plugin and potentially third parties.

Reply:
Copy pasta. Fixed in https://github.com/facebookincubator/facebook-for-woocommerce/pull/1539/commits/a7b5f951a3f87916a0f4f7e110aa61033208efda.